plugins/infinispan90/src/main/java/org/radargun/service/Infinispan90Lifecycle.java

```diff
@@ -0,0 +1,24 @@
+package org.radargun.service;
+
+import org.infinispan.remoting.transport.jgroups.JGroupsTransport;
+import org.jgroups.JChannel;
+
+/**
+ * @author Roman Macor (rmacor@redhat.com)
+ */
+public class Infinispan90Lifecycle extends Infinispan52Lifecycle {
+
+   public Infinispan90Lifecycle(Infinispan90EmbeddedService service) {
+      super(service);
+   }
+
+   @Override
+   protected JGroupsTransport getTransport() {
```

Comment:
Nope. @alanfx please integrate when @rmacor confirms that it will work with JGroups 3.6 which are still in master.